src/bindings/python/tests/test_runtime/test_properties.py

```diff
@@ -28,7 +28,7 @@ def test_properties_ro_base():
 
 
 def test_properties_rw_base():
-    assert props.cache_dir() == "CACHE_DIR"
+    assert props.cache_dir == "CACHE_DIR"
```

Comment:
```suggestion
    assert ov.properties.cache_dir == "CACHE_DIR"
```
Can one or two "longer" imports be checked?

Reply:
added